### ISSUE TITLE
Why is animation disabled in zoom other than 1?

### ISSUE THREAD

**Issue by hnuecke** [blasten/turn.js#327]
The page flip animation only works with zoom=1.
If zoom is set to lower or higher numbers, the start event is supressed with event.preventDefault()

Why??

**Comment by alQemist** [comment]
Seems to make sense to me. I can imagine that the animation would produce some weird results with the page zoomed in to 2x. What do you have in mind ?

**Comment by hnuecke** [comment]
I agree that for zoom > 1 this probably is not a good idea. But if zoomed out I would expect I can operate the book as with zoom=1.
If e.g. the page size is too big to fit on a small screen, you could zoom out to make it fit. And still see the animation. nice.

**Comment by alQemist** [comment]
I would suggest you set the magazine size to fit the users' particular screen upon initializing. That way the user has to do nothing.

**Comment by hnuecke** [comment]
That's probably the best option; keep it simple ;-)
The reason I was thinking about a zoom option was because that's what the alternative Flash viewer is offering. This allows the user to freely adjust the size, so it is up to him to find the best compromise between best readability (filling the screen completely) and best support of the page flip effect, i.e. while a page is dragged. 
A better option could be to scale the book giving sufficient room for the effect, and then let the user zoom with the standard zoom features availble for the device. And making use of zoomIn and zoomOut.

Thaks for the discussion!

**Comment by alQemist** [comment]
Hey you could make the Div container for your book "resizable" the user could do what you describe and turn.js will handle the resizing automatically if you have added your own code to define the magazine width/height

**Comment by hnuecke** [comment]
OK, I'll give it a try and will "play" a bit with the different options.
What worries me a bit are the browser differences, e.g. when using %; and the different strategies regarding viewport settings.
Thanks for your helpful feedback!

**Comment by alQemist** [comment]
I have to day Emmanuel did a beautiful job with turn.js - so far it seems totally compatible with all the major browsers :)

**Comment by hnuecke** [comment]
I Fully agree
